### PR TITLE
Print warnings occurred during commit

### PR DESCRIPTION
**Pull request by dmitry-starodubov**

### CODE REVIEW

File: src/isql/isql.epp [hunk 1]
```diff
@@ -244,6 +244,7 @@ namespace IcuUtil
 	}
 }
 
+void ISQL_warning(Firebird::CheckStatusWrapper* st);
```
Comment (aafemt) [inline src/isql/isql.epp:247]: It would be better to fix declaration in isql_proto.h.

Comment (dmitry-starodubov) [reply src/isql/isql.epp:247]: Indeed. Didn't see it there.

Comment (AlexPeshkoff) [reply src/isql/isql.epp:247]: @dmitry-starodubov Will you apply changes, suggested by DS, or I can do it myself?

Comment (dmitry-starodubov) [reply src/isql/isql.epp]: Done

**Pull request merged**